plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/utils.ts

```diff
@@ -0,0 +1,28 @@
+export const getAdditionalData = (content: string | undefined, format: string) => {
```

Comment:
```suggestion
export const getAdditionalCSVData = (content: string | undefined, format: string) => {
```
I guess we should name it as CSV to avoid confusing.